thesis-api/main/services/question_service/definition_service.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+import wikipedia as wikipedia
+
+
+class DefinitionService(ABC):
+    @abstractmethod
+    def get_definition(self, word: str) -> str:
+        pass
+
+
+class WikipediaDefinitionService(DefinitionService):
+    def get_definition(self, word_to_find: str) -> tuple[str, str]:
+        word = wikipedia.search(word_to_find)[0]
```

Comment:
What happens if the word has multiple meanings and there are multiple articles in Wikipedia?